src/Illuminate/Auth/EloquentUserProvider.php

```diff
@@ -68,7 +68,7 @@ public function retrieveByToken($identifier, $token)
             return null;
         }
 
-        $rememberToken = $model->getRememberToken();
+        $rememberToken = (string) $model->getRememberToken();
```

Comment:
I don't think this is the correct fix. Surely the correct fix is to fix `getRememberToken` to always return a string?